Mathlib/Computability/AkraBazzi/SumTransform.lean

```diff
@@ -12,12 +12,16 @@ import Mathlib.Analysis.SpecialFunctions.Log.Deriv
 /-!
 # Akra-Bazzi theorem: the sum transform
 
-We develop further required preliminaries for the theorem, up to the sum transform.
+We develop further preliminaries required for the theorem, up to the sum transform.
 
 ## Main definitions and results
 
 * `AkraBazziRecurrence T g a b r`: the predicate stating that `T : ℕ → ℝ` satisfies an Akra-Bazzi
   recurrence with parameters `g`, `a`, `b` and `r` as above.
+* `smoothingFn`: the smoothing function `ε(x) = 1 / log x` (with derivative/asymptotic
```

Comment:
Can you move this part to a separate PR? I cannot check this easily from the diff.